rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/01/27"
+maturity = "production"
+updated_date = "2021/01/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies modification of the dynamic linker preload shared object (ld.so.preload). Adversaries may execute malicious payloads by hijacking the dynamic linker used to load libraries.
```

Comment:
re lint

Reply:
done, all rules will be relinted prior to merging anyway.